src/state/KindeProvider.tsx

```
@@ -1,256 +1,409 @@
+        return;
+      }
 
-  const getStringFlag = useCallback(
-    (code: string, defaultValue: string) =>
-      client?.getStringFlag(code, defaultValue) || defaultValue,
-    [client]
-  );
+      const optionsState: Record<string, string> = options.state || {};
 
-  const getPermissions = useCallback(() => client?.getPermissions(), [client]);
+      delete options.state;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)